http-builder-ng-apache/src/test/groovy/groovyx/net/http/ApacheHttpPatchSpec.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2017 David Clark
```

Comment:
I will have an update to the copyright headers going in shortly... changing it to "HttpBuilder-NG Project" rather than Dave specifically (he approved). Sorry, but this may result in some conflicts that you will need to resolve on your end.